docs/docs/company/contact-us.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 4
+---
+
+# Contact Us
+
+## Customers, Users, and Future Users
+
+- For any feedback or questions, please email us at [contact@stoat.dev](mailto:contact@stoat.dev).
+- For bug reports or feature suggestions, please [create an issue on GitHub](https://github.com/stoat-dev/stoat-action/issues/new).
+- If you prefer video calls, just [choose a time](https://calendly.com/stoat-dev/30-min). We'd love to chat!
+
+## Investors
+
+- We are planning on raising money in early 2023.
+- Unless we have already agreed to chat, we will decline your call if you use our Calendly link above.
+- Please let us know you are interested by emailing us at [contact@stoat.dev](mailto:contact@stoat.dev) or [adding Jared on LinkedIn](https://www.linkedin.com/in/jrhizor/).
+- We'll reach out to you when we're ready to connect.
```

Comment:
Nitpick. The second bullet point seems a bit arrogant?

Maybe merge the second bullet point with the last one and tune down a little bit:

```suggestion
- Currently, we are prioritizing meetings with Stoat customers and users, and will likely reject video calls from investors if you use our Calendly link above. We will reach out to you as soon as possible when we're ready to connect.
```

Reply:
I'll do that except remove the word "likely"